src/main/resources/db/migration/V002__job.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE JOB_ENTITY (
```

Comment:
You deleted ExampleEntity related code. So in this migration file, you should drop `EXAMPLE_ENTITY` table.

Reply:
OK. Thinking now, the first migration file shouldn't be deleted? Because the `EXAMPLE_ENTITY` table will never go to production environment.

Reply:
In a real scenario, we must not delete migration files. Flyt would try to validate the version `001` against database, but this version wouldn't exist in our code.
As this project here is in its inicial phase and it's not on production yet, so it's up to you: either you delete the `V001` migration file and then you rename this one to be the new `V001`, or you add the `DROP TABLE` here (in version 002).